### PR TITLE
Fix CI problems

### LINKED ISSUE #79: ci/cd: fix inconsistent linter

### PR DESCRIPTION
**Pull request by bdmendes**
Closes #79 

Removes audit from PR, relaxes Prettier and fixes Netlify deploy issues.

# Screenshots

# Review checklist

- [ ] Contains enough appropriate tests
- [ ] Behavior is as expected
- [ ] Clean, well-structured code

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -73,6 +73,7 @@ jobs:
   audit:
     runs-on: ubuntu-latest
     name: Audit
+    if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop'
```
Comment (BrunoRosendo) [inline .github/workflows/ci.yml]: What do you think of extracting the audit to another file and using the "on master pull_request" option? I think we could avoid the weird if by doing that

**Review by BrunoRosendo — CHANGES_REQUESTED** [review]
Good idea to improve development quality. I just left a cosmetic suggestion

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bdmendes) [reply .github/workflows/ci.yml]: Sure, I'll make the change

**Review by ttoino — APPROVED** [review]
LGTM!

**Review by jamcunha — APPROVED** [review]
LGTM

**Review by BrunoRosendo — APPROVED** [review]

**Pull request merged**